<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Title</class>
 <widget class="QMainWindow" name="Title">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1295</width>
    <height>936</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ropa</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="mainLayout" stretch="2,4,3">
      <item>
       <layout class="QVBoxLayout" name="favouritesPane">
        <item>
         <layout class="QHBoxLayout" name="favouritesPadding">
          <property name="bottomMargin">
           <number>10</number>
          </property>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="favouritesLabel">
          <property name="text">
           <string>Favourites</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="favouritesList"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="chainPane">
        <item>
         <layout class="QHBoxLayout" name="chainPadding">
          <property name="bottomMargin">
           <number>10</number>
          </property>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="chainLabel">
          <property name="text">
           <string>ROP Chain</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="chainList">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="searchPane" stretch="0,0,0,0,0,0,0">
        <item>
         <layout class="QHBoxLayout" name="searchPadding_1">
          <property name="bottomMargin">
           <number>10</number>
          </property>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="searchTitle">
          <property name="text">
           <string>Search</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="searchPadding_0">
          <property name="bottomMargin">
           <number>6</number>
          </property>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="searchConditions">
          <item>
           <layout class="QVBoxLayout" name="searchInputLabels">
            <item>
             <widget class="QLabel" name="badbytesLabel">
              <property name="font">
               <font>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="text">
               <string> BadBytes: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="searchLabel">
              <property name="font">
               <font>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="text">
               <string> Filter: </string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="searchInputBoxes">
            <item>
             <widget class="QLineEdit" name="badbytesInput">
              <property name="font">
               <font>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="text">
               <string>0a0d</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="searchBar">
              <property name="font">
               <font>
                <pointsize>9</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="searchPadding_2">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="searchButtons">
          <item>
           <widget class="QPushButton" name="searchInstructions">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Instructions</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <property name="autoRepeat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="searchPopPopRet">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>PopPopRet</string>
            </property>
            <property name="autoRepeat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="searchSemantics">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Semantics</string>
            </property>
            <property name="autoRepeat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="searchList">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1295</width>
     <height>44</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuVIew">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport_as">
     <property name="title">
      <string>Export as</string>
     </property>
     <addaction name="actionStruct"/>
     <addaction name="actionPwntools"/>
    </widget>
    <widget class="QMenu" name="menuRecent">
     <property name="title">
      <string>Open Recent</string>
     </property>
     <addaction name="itemRecent0"/>
     <addaction name="itemRecent1"/>
     <addaction name="itemRecent2"/>
     <addaction name="itemRecent3"/>
     <addaction name="itemRecent4"/>
     <addaction name="itemRecentEmpty"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="menuRecent"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="menuExport_as"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuVIew"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New Project...</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open Project...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionStruct">
   <property name="text">
    <string>Python (struct)</string>
   </property>
  </action>
  <action name="actionPwntools">
   <property name="text">
    <string>Python (pwntools)</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="itemRecent0">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="itemRecent1">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="itemRecent2">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="itemRecent3">
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="itemRecent4">
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="itemRecentEmpty">
   <property name="text">
    <string/>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
